geode-core/src/test/java/org/apache/geode/internal/cache/control/SerializableRestoreRedundancyResultsImplTest.java

```diff
@@ -153,39 +157,64 @@ public void addRegionResultsAddsToCorrectInternalMapAndAddsPrimaryReassignmentDe
     results.addRegionResults(regionResults);
 
     Map<String, RegionRedundancyStatus> zeroRedundancyResults =
         results.getZeroRedundancyRegionResults();
-    assertThat(zeroRedundancyResults.size(), is(1));
-    assertThat(zeroRedundancyResults.get(zeroRedundancyRegionName), is(zeroRedundancyRegionResult));
+    assertThat(zeroRedundancyResults.size()).isEqualTo(1);
+    assertThat(zeroRedundancyResults.get(zeroRedundancyRegionName))
+        .isEqualTo(zeroRedundancyRegionResult);
 
     Map<String, RegionRedundancyStatus> underRedundancyResults =
         results.getUnderRedundancyRegionResults();
-    assertThat(underRedundancyResults.size(), is(1));
-    assertThat(underRedundancyResults.get(underRedundancyRegionName),
-        is(underRedundancyRegionResult));
+    assertThat(underRedundancyResults.size()).isEqualTo(1);
+    assertThat(underRedundancyResults.get(underRedundancyRegionName))
+        .isEqualTo(underRedundancyRegionResult);
 
     Map<String, RegionRedundancyStatus> successfulRegionResults =
         results.getSatisfiedRedundancyRegionResults();
-    assertThat(successfulRegionResults.size(), is(1));
-    assertThat(successfulRegionResults.get(successfulRegionName), is(successfulRegionResult));
+    assertThat(successfulRegionResults.size()).isEqualTo(1);
+    assertThat(successfulRegionResults.get(successfulRegionName)).isEqualTo(successfulRegionResult);
 
-    assertThat(results.getTotalPrimaryTransfersCompleted(), is(transfersCompleted));
-    assertThat(results.getTotalPrimaryTransferTime().toMillis(), is(transferTime));
+    assertThat(results.getTotalPrimaryTransfersCompleted()).isEqualTo(transfersCompleted);
+    assertThat(results.getTotalPrimaryTransferTime()).isEqualTo(transferTime);
   }
 
   @Test
   public void addPrimaryDetailsUpdatesValue() {
-    assertThat(results.getTotalPrimaryTransfersCompleted(), is(0));
-    assertThat(results.getTotalPrimaryTransferTime().toMillis(), is(0L));
+    assertThat(results.getTotalPrimaryTransfersCompleted()).isEqualTo(0);
+    assertThat(results.getTotalPrimaryTransferTime()).isEqualTo(0L);
     results.addPrimaryReassignmentDetails(details);
-    assertThat(results.getTotalPrimaryTransfersCompleted(), is(transfersCompleted));
-    assertThat(results.getTotalPrimaryTransferTime().toMillis(), is(transferTime));
+    assertThat(results.getTotalPrimaryTransfersCompleted()).isEqualTo(transfersCompleted);
+    assertThat(results.getTotalPrimaryTransferTime()).isEqualTo(transferTime);
     results.addPrimaryReassignmentDetails(details);
-    assertThat(results.getTotalPrimaryTransfersCompleted(), is(transfersCompleted * 2));
-    assertThat(results.getTotalPrimaryTransferTime().toMillis(), is(transferTime * 2));
+    assertThat(results.getTotalPrimaryTransfersCompleted()).isEqualTo(transfersCompleted * 2);
+    assertThat(results.getTotalPrimaryTransferTime()).isEqualTo(transferTime * 2);
+  }
+
+  @Test
+  public void testSerializable() throws JsonProcessingException {
```

Comment:
This test sets multiple fields but does not verify all of them. Would it be possible to confirm that all data in the object is correctly serialized and deserialized, not just the message and status fields?

Reply:
done. assertThat(value).isEqualToComparingFieldByField(restoreRedundancyResults);